jans-config-api/plugins/user-mgt-plugin/src/main/java/io/jans/configapi/plugin/mgt/rest/UserResource.java

```diff
@@ -83,8 +83,8 @@ public Response getUsers(
             @Parameter(description = "Attribute whose value will be used to order the returned response") @QueryParam(value = ApiConstants.SORT_BY) String sortBy,
             @Parameter(description = "Order in which the sortBy param is applied. Allowed values are \"ascending\" and \"descending\"") @QueryParam(value = ApiConstants.SORT_ORDER) String sortOrder)
             throws IllegalAccessException, InvocationTargetException {
-        if (logger.isDebugEnabled()) {
-            logger.debug("User search param - limit:{}, pattern:{}, startIndex:{}, sortBy:{}, sortOrder:{}",
+        if (logger.isInfoEnabled()) {
+            logger.info("User search param - limit:{}, pattern:{}, startIndex:{}, sortBy:{}, sortOrder:{}",
```

Comment:
@pujavs We are flipping logging level back on forth all the time. I guess we should decide what level do we use to log request/response and simply stick to it.

Reply:
Apologies, there are issues cropping up for user mgmt for different persistence type and needed the logs to appear for these. Since debug is not the default log level and these logs were not appearing in logs when reported.